Misc/NEWS.d/next/Windows/2021-05-02-15-29-33.bpo-43984.U92jiv.rst

```diff
@@ -0,0 +1,3 @@
+:meth:`winreg.SetValueEx` now leaves the target register untouched in the case of conversion errors.
```

Comment:
You mean "target value", not "target register". In computing, a "register" is a unit of memory in a processor.

Reply:
Oops. I'll fix that tomorrow.